### PR TITLE
Update doc to mention debug mode

### PR DESCRIPTION
**Pull request by crr0004**
Would be good to have a mention of the debug mode in the docs.

Submission is just my knowledge of it. Not sure of the levels that are being used. Not too sure of the documentation formatting so feel free to change that if it's wrong. Don't usually write documentation for VIM plugins

### CODE REVIEW

File: doc/clang.txt [hunk 1]
```diff
@@ -86,6 +86,11 @@ g:clang_cpp_completeopt                 *g:clang_cpp_completeopt*
         " Have preview window for C++ as its complex prototypes
         let g:clang_cpp_completeopt = 'longest,menuone,preview'
 <
+g:clang_debug                           *g:clang_debug*
+        Turns on debug mode with value being verbosity
+        Default: >
+            let g:clang_debug = 1
```
Comment (justmao945) [inline doc/clang.txt]: default is disabled `let g:clang_debug = 0`

Comment (justmao945) [inline doc/clang.txt]: You can add valid values: 1 2 3 ...

Comment (crr0004) [inline doc/clang.txt]: Cheers. Fixed the default and added information about the values.

Am I right in saying the max input value for s:PDebug so far is 3?

**Comment by justmao945** [comment]
thx 😄

**Pull request merged**